edk2toolext/edk2_logging.py

```diff
@@ -255,6 +255,13 @@ def __init__(self):
         self._verbose = False
         self._currentSection = "root"
 
+        secrets_regex_strings = [
+            r"[A-Za-z0-9]{46}",  # Nuget API Key
+            r"github_pat_[^ ]*"  # Github PAT
```

Comment:
Might also be worth considering using `+` instead of `*` since a prefix without any matched characters doesn't need to be hidden.

Could also use `{<min>,}` if the min token length ever used is documented somewhere. But, I think `+` would help differentiate a little better between a real need to hide the value and something that might unnecessarily impact debuggability.

Reply:
Going with + over * and {<min>, 255} as I cant find anything about a min value, only that the max is 255. (I didin't look hard, and I think + is better anyway.)